functions/Get-DbaUserPermission.ps1

```diff
@@ -200,11 +200,15 @@ function Get-DbaUserPermission {
                 Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
 
-            $dbs = $server.Databases
+            $dbs = $server.Databases | Where-Object { $_.IsAccessible -eq $true }
```

Comment:
With that change, we don't need the `if ($db.IsAccessible -eq $false) {` down in line 230.